bindings/python/python/opendal/__init__.pyi

```diff
@@ -23,7 +23,16 @@ class Operator:
     def __init__(self, scheme: str, **kwargs): ...
     def read(self, path: str) -> bytes: ...
     def open_reader(self, path: str) -> Reader: ...
-    def write(self, path: str, bs: bytes): ...
+    def write(
+        self,
+        path: str,
+        bs: bytes,
+        append: bool = None,
+        buffer: int = None,
+        content_type: str = None,
+        content_disposition: str = None,
+        cache_control: str = None,
```

Comment:
Hey, cc @messense, is it okay to use `kwargs` in the API and then expand them in the `pyi` files?

Reply:
Yes, should be fine.